### PR TITLE
BE-226 - Additional refinement is needed regarding merchants and merchant category codes

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Refined definitions to address feedback on FBC-260 with respect to merchants and merchant codes, clarifying definitions, adding notes, and eliminating a redundant property

Fixes: #1175 / BE-226

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](../CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](../ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Review by dsnewman — DISMISSED** [review]

**Review by rivettp — CHANGES_REQUESTED** [review]
The definition of Merchant is imprecise and contradicted by its explanatoryNote. 
To be honest I'm not sure why FIBO needs it - if it's just for the domain of a MerchantCode then make that the basis of the definition - since it's outside the concern of FIBO to determine which businesses are eligible for a code. You could just define it as "business that makes sales and is eligible to accept payments by payment card or equivalent".
I think the area of Merchant Code Schemes is a lot more complex than indicated - and could involve jurisdiction or organization-specific extensions and mappings between those and Transaction Codes. Whether we care about that is unclear: we don't have a well-defined use case. If we want people to be able to search for card products by what categories they provide extra spending points then we probably have enough and we would not even need Merchant. If we ALSO want to be able to search against specific merchants where they could receive such extra points, then we'd need more, but IMO that would make it more of a Retail ontology.

File: BE/FunctionalEntities/FunctionalEntities.rdf [hunk 1]
```diff
@@ -129,11 +132,14 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>merchant</rdfs:label>
-		<skos:definition>party engaged in the purchase and sales of goods for profit</skos:definition>
+		<skos:definition>party engaged in the purchase and sales of goods produced by others for profit</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom>https://dictionary.cambridge.org/dictionary/english/merchant</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:adaptedFrom>https://www.merriam-webster.com/thesaurus/merchant</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>The U.S. Uniform Commercial Code (UCC, section 2-104) defines a merchant as a party that &apos;deals in goods of the kind or otherwise by his occupation holds himself out as having knowledge or skill peculiar to the practices or goods involved in the transaction or to whom such knowledge or skill may be attributed by his employment of an agent or broker or other intermediary who by his occupation holds himself out as having such knowledge or skill&apos;.</fibo-fnd-utl-av:explanatoryNote>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply BE/FunctionalEntities/FunctionalEntities.rdf:138]: I'll remove the explanatory note.  The definition is exactly what you find in almost every dictionary you look at, however.  TA farmer is not a merchant unless they purchase food items from other farmers and sell them.  In their role of purchasing something someone else produced and reselling it they could be a merchant, but not just selling their own stuff at say a farmer's market.  A craftsperson selling things they have made is not a merchant by the same definition.  Consultants are not merchants.  Day traders are trading on their own behalf and are therefore not merchants.

File: BE/FunctionalEntities/FunctionalEntities.rdf [hunk 2]
```diff
@@ -173,12 +171,12 @@
...
+		<fibo-fnd-utl-av:explanatoryNote>Values are specified only for those merchant categories that are generally expected to originate retail financial transactions.  Criteria for establishing a new category code includes (a) the merchant category is reasonable and substantially different from all other merchant categories currently represented in the list of code values; (b) the merchant category is separate and distinct from all other industries currently represented in the list of code values; (c) the proposal describes a merchant category or industry, and not a process; (d) the minimum annual sales volume of merchants included in the merchant category, taken as a whole, is USD 10 million; and (e) sufficient justification for the addition of a new code value is found.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-fct-fct;MerchantCategoryCodeScheme">
-		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Arrangements/ClassificationSchemes/IndustrySectorClassificationScheme"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;IndustrySectorClassificationScheme"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply BE/FunctionalEntities/FunctionalEntities.rdf:179]: No - they all use the ISO standard.  Mastercard may extend it, but the processors that WF and other banks use to process credit cards via these networks use the ISO codes, not only for consumer cards but for wholesale credit card activities.

This is something that is essential for not only the generic credit card content for schema.org but for potential use by retail as well, who also want this.

**Review by rivettp — APPROVED** [review]
OK, but I remain concerned that we're continuing to make extensions without any semblance of use cases, competency questions, examples or tests.

**Review by trypuz — APPROVED** [review]
Approved

**Pull request merged**